hledger-lib/hledger-lib.cabal

```diff
@@ -78,9 +78,11 @@ library
     , mtl
     , mtl-compat
     , old-time
-    , parsec >= 3
+    , megaparsec >= 5
+    , parsec
     , regex-tdfa
     , safe >= 0.2
+    , semigroups
```

Comment:
It is used in Hledger.Read.Common.hs in Data.List.NonEmpty. It moved to base in ghc 8, but you probably want to support older GHCs as well.